lua/kubectl/utils/find.lua

```diff
@@ -25,7 +25,7 @@ end
 ---@param tbl table
 ---@param str string
 ---@return boolean
-function M.is_in_table(tbl, str, exact)
+function M.is_in_table(tbl, str)
```

Comment:
Are we not using exact here?: https://github.com/Ramilito/kubectl.nvim/blob/e0bfb6eda2248e5f4ef83cf35cca44e95645e253/lua/kubectl/views/init.lua#L293

Reply:
True. My apologies. Not sure what happened to tooling last night, will look into why and fix.

Reverted.

I did notice that the `exact` param is not being passed to the recursive call, though, resulting in inconsistent `exact` matching with nested tables. What do you think?

Reply:
we had problems with it:
https://github.com/Ramilito/kubectl.nvim/issues/308

Reply:
changes to explicitly pass false for recursive calls